2021-12-06T22:07:11.896143627Z stderr F I1206 22:07:11.895807       1 autoscaler.go:49] Scaling Namespace: kube-system, Target: deployment/coredns
2021-12-06T22:07:12.236045906Z stderr F I1206 22:07:12.235837       1 plugin.go:50] Set control mode to ladder
2021-12-06T22:07:12.236102106Z stderr F I1206 22:07:12.235871       1 ladder_controller.go:72] Detected ConfigMap version change (old:  new: 649) - rebuilding lookup entries
2021-12-06T22:07:12.236115306Z stderr F I1206 22:07:12.235879       1 ladder_controller.go:73] Params from apiserver: 
2021-12-06T22:07:12.236125506Z stderr F {"coresToReplicas":[[1,2],[512,3],[1024,4],[2048,5]],"nodesToReplicas":[[1,2],[8,3],[16,4],[32,5]]}
2021-12-06T22:15:37.5541623Z stderr F E1206 22:15:37.553938       1 reflector.go:125] github.com/kubernetes-incubator/cluster-proportional-autoscaler/pkg/autoscaler/k8sclient/k8sclient.go:96: Failed to list *v1.Node: Get https://davidscaletest-dns-305d7c8d.hcp.westus2.azmk8s.io:443/api/v1/nodes: dial tcp 52.143.92.12:443: connect: connection refused
2021-12-06T22:15:51.196796085Z stderr F E1206 22:15:51.196581       1 reflector.go:125] github.com/kubernetes-incubator/cluster-proportional-autoscaler/pkg/autoscaler/k8sclient/k8sclient.go:96: Failed to list *v1.Node: Get https://davidscaletest-dns-305d7c8d.hcp.westus2.azmk8s.io:443/api/v1/nodes: dial tcp 52.143.92.12:443: connect: connection refused
2021-12-06T22:15:51.995984773Z stderr F I1206 22:15:51.995771       1 autoscaler_server.go:157] ConfigMap not found: Get https://davidscaletest-dns-305d7c8d.hcp.westus2.azmk8s.io:443/api/v1/namespaces/kube-system/configmaps/coredns-autoscaler: dial tcp 52.143.92.12:443: connect: connection refused, will create one with default params
2021-12-06T22:15:52.213259932Z stderr F E1206 22:15:52.212992       1 reflector.go:125] github.com/kubernetes-incubator/cluster-proportional-autoscaler/pkg/autoscaler/k8sclient/k8sclient.go:96: Failed to list *v1.Node: Get https://davidscaletest-dns-305d7c8d.hcp.westus2.azmk8s.io:443/api/v1/nodes: dial tcp 52.143.92.12:443: connect: connection refused
2021-12-06T22:15:52.621790068Z stderr F E1206 22:15:52.621651       1 autoscaler_server.go:117] Error syncing configMap with apiserver: configmaps "coredns-autoscaler" already exists
